src/ipyradiant/__init__.py

```diff
@@ -4,6 +4,21 @@
 from ._version import __version__
 from .loader import LoadWidget
 from .query import QueryWidget
-from .visualization import CytoscapeVisualization
+from .visualization import (
+    CytoscapeVisualization,
```

Comment:
Just noticed this tiny pattern inconsitency too. Lets settle on a common Visualization class pattern. 

Either `*Vis` or `*Visualization` or maybe middle ground `*Visualizer`.